dapp/config/default.json

```diff
@@ -39,32 +39,23 @@
         "RupiahToken": "0x998FFE1E43fAcffb941dc337dD0468d52bA5b48A"
       },
       "fuse": {
-        "ropsten": {
```

Comment:
we can also do it like this:
```
fuse: {
  multiBridge: {
     ropsten: 0x'',
     main: '0x'
}
```
But I prefer that children of adressess will be networks. like: `addresses.fuse.contract`